### PR TITLE
Add todo docs

### PR DESCRIPTION
**Pull request by beth-panx**

### CODE REVIEW

File: concepts/toolkit/components/todo.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| Event | Detail | Description |
+| --- | --- | --- |
+| taskAdded | The detail contains the respective `task` object | Fires when a new task has been created. |
+| taskChanged | The detail contains the respective `task` object | Fires when task metadata has been changed, such as marking completed. |
+| taskClick | The detail contains the respective `task` object | Fires when the user clicks or taps on a task. |
+| taskRemoved | The detail contains the respective `task` object | Fires when an existing task has been deleted. |
+
+## Templates
+
+The `tasks` component supports several [templates](../templates.md) that allow you to replace certain parts of the component. To specify a template, include a `<template>` element inside a component and set the `data-type` value to one of the following:
```
Comment (nmetulev) [inline concepts/toolkit/components/todo.md:97]: The templates link here should be updated to the new location

**Review by nmetulev — CHANGES_REQUESTED** [review]
Awesome

Make sure to add this to the toc.yaml so it shows up in the navigation pane on the left.

File: concepts/toolkit/components/todo.md (hunk 1 above)
Comment (beth-panx) [reply concepts/toolkit/components/todo.md:97]: Is there still a template md file?

File: concepts/toolkit/components/todo.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+title: "To-do component in the Microsoft Graph Toolkit"
+description: "The To-do component enables the user to view, add, remove, complete, or edit todo tasks. It works with any tasks in Microsoft To-Do."
+localization_priority: Normal
+author: shweaver-MSFT
+---
+
+# To-do Component in the Microsoft Graph Toolkit
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:8]:
```suggestion
To Do Component in the Microsoft Graph Toolkit
```

File: concepts/toolkit/components/todo.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+The To-do component enables the user to view, add, remove, complete, or edit tasks. It works with tasks from the Microsoft To-Do API.
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:10]:
```suggestion
The To Do component is used to enable the signed-in user to view, add, remove, complete, and/or edit tasks from Microsoft To Do using the To Do tasks API on Microsoft Graph.
```

File: concepts/toolkit/components/todo.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+author: shweaver-MSFT
+---
+
+# To-do Component in the Microsoft Graph Toolkit
+
+The To-do component enables the user to view, add, remove, complete, or edit tasks. It works with tasks from the Microsoft To-Do API.  
+
+## Example
+
+The following example displays the signed-in user's Microsoft To-do tasks using the `mgt-todo` component. You can use the code editor to see how [properties](#properties) change the behavior of the component.
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:14]:
```suggestion
The following example displays the signed-in user's Microsoft To Do tasks using the `mgt-todo` component. You can use the code editor to see how [properties](#properties) change the behavior of the component.
```

File: concepts/toolkit/components/todo.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+<iframe src="https://mgt.dev/iframe.html?id=components-mgt-todo--tasks&source=docs" height="500"></iframe>
+
+[Open this example in mgt.dev](https://mgt.dev/?path=/story/components-mgt-todo--tasks&source=docs)
+
+## Properties
+
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:21]:
```suggestion

You can use the following attributes and properties to customize the component.

```

File: concepts/toolkit/components/todo.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+| N/A | taskFilter  | An optional function to filter which tasks are shown to the user. |
+
+The following example shows only tasks from the folder with ID *12345* and does not allow the user to create new tasks.
+
+```html
+<mgt-todo read-only initial-id="12345"></mgt-todo>
+```
+
+## Custom CSS variables
+
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:39]:
```suggestion

The `mgt-todo` component defines the following CSS custom properties.

```

File: concepts/toolkit/components/todo.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  --task-icon-background
+  --task-icon-background-completed
+  --task-icon-border
+  --task-icon-border-completed
+  --task-icon-border-radius
+  --task-icon-color
+  --task-icon-color-completed
+}
+````
+
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:86]:
```suggestion

To learn more, see [styling components](https://docs.microsoft.com/graph/toolkit/style.md).

```

File: concepts/toolkit/components/todo.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+## Events
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:87]:
```suggestion
## Events

The following events are fired from the component.

```

File: concepts/toolkit/components/todo.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+        </div>
+    </template>
+</mgt-todo>
+```
+
+## Microsoft Graph permissions
+
+This control uses the following Microsoft Graph APIs and permissions.
+
+| Resource | Permission |
```
Comment (elisenyang) [inline concepts/toolkit/components/todo.md:120]: These should link to the API reference page for the endpoints

**Review by elisenyang — DISMISSED** [review]

File: concepts/toolkit/components/todo.md (hunk 9 above)
Comment (beth-panx) [reply concepts/toolkit/components/todo.md:129]: awww.... no suggested commit? Lol I will update this.

Comment (beth-panx) [reply concepts/toolkit/components/todo.md:129]: Done! @elisenyang

File: concepts/toolkit/components/todo.md (hunk 1 above)
Comment (beth-panx) [reply concepts/toolkit/components/todo.md]: @nmetulev updated!

File: concepts/toolkit/components/todo.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+| - | - |
+| [/me/todo/lists/](https://docs.microsoft.com/en-us/graph/api/todo-list-lists?view=graph-rest-1.0&tabs=http) | Tasks.ReadWrite |
+| [/me/todo/lists/{todoTaskListId}/tasks](https://docs.microsoft.com/en-us/graph/api/todotasklist-list-tasks?view=graph-rest-1.0&tabs=http) | Tasks.ReadWrite |
+| [/me/todo/lists/{todoTaskListId}/tasks/{taskId}](https://docs.microsoft.com/en-us/graph/api/todotask-get?view=graph-rest-1.0&tabs=http) | Tasks.ReadWrite |
```
Comment (nmetulev) [inline concepts/toolkit/components/todo.md:133]:
```suggestion
| [/me/todo/lists/](/graph/api/todo-list-lists) | Tasks.ReadWrite |
| [/me/todo/lists/{todoTaskListId}/tasks](/graph/api/todotasklist-list-tasks) | Tasks.ReadWrite |
| [/me/todo/lists/{todoTaskListId}/tasks/{taskId}](/graph/api/todotask-get) | Tasks.ReadWrite |
```

**Review by nmetulev — CHANGES_REQUESTED** [review]
Need to add the todo to the toc.yaml file so it shows up in the nav

Also add a link to the todo docs from the overview page

**Review by nmetulev — APPROVED** [review]

**Pull request merged**